src/ui/components/DismissibleTextForm/index.js

```diff
@@ -81,9 +84,11 @@ export class DismissibleTextFormBase extends React.Component<
   };
 
   onDismiss = (event: SyntheticEvent<any>) => {
+    const { onDismiss } = this.props;
     event.preventDefault();
+    invariant(onDismiss, 'onDismiss() is required');
```

Comment:
would this even be needed if the button itself is wrapped around the onDimiss?

Reply:
Flow can't follow that kind of logic so it needs some kind of protection against falsy values here. Aside from appeasing Flow, showing an error like this might be helpful if someone accidentally exposed this callback after refactoring the code or something.